packages/app/src/app/pages/SignIn/Onboarding.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import { Element, Text, Stack } from '@codesandbox/components';
+import { useAppState, useActions } from 'app/overmind';
+import { InputText } from 'app/components/dashboard/InputText';
+import { InputSelect } from 'app/components/dashboard/InputSelect';
+import { Button } from 'app/components/dashboard/Button';
+
+const roleOptions = [
```

Comment:
nitpick: can you use UPPER_CASE here to make it clear these are constant values? not some dynamic / state provided object